mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/KeyRegistrationHandler.java

```diff
@@ -78,10 +78,10 @@
     }
 
     void start() {
-        sender.declareQueue(QueueSpecification.queue()
+        sender.declareQueue(QueueSpecification.queue(eventBusId.asString())
             .durable(DURABLE)
-            .exclusive(EXCLUSIVE)
-            .autoDelete(AUTO_DELETE)
+            .exclusive(!EXCLUSIVE)
```

Comment:
Maybe having `exclusive` and `notExclusive` would be better, I'm not fan of `bool`

Reply:
We avoid negative naming cf @mbaechler 

A local boolean operation is considered more explicit...

Reply:
agreed then